### PR TITLE
boilerplate update

### PR DESCRIPTION
**Pull request by a7vicky**

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -66,14 +65,13 @@ aliases:
     - reedcort
     - mrWinston
   srep-functional-leads:
-    - mrbarge
     - rafael-azevedo
     - iamkirkbater
     - bng0y
     - tonytheleg
```
Comment (weherdh) [inline OWNERS_ALIASES:71]: Should add `ravitri` to the `srep-funtional-leads`?

Comment (ravitri) [reply OWNERS_ALIASES:71]: Thanks Wenqi! This change will need to be part of boilerplate - https://github.com/openshift/boilerplate/blob/master/boilerplate/openshift/golang-osd-operator/OWNERS_ALIASES#L67 

I'll raise a PR for that change. Thanks to notice it.

**Comment by ravitri** [comment]
/hold

Let's wait for https://github.com/openshift/boilerplate/pull/312 to be merged and then rebase this PR and then merge.

**Pull request merged**